### PR TITLE
[15.0][MIG] auth_oauth_multi_token: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36441

### CODE REVIEW

File: auth_oauth_multi_token/tests/test_multi_token.py [hunk 1]
```diff
@@ -4,28 +4,24 @@
...
-    at_install = False
-
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.token_model = cls.env["auth.oauth.multi.token"]
-        cls.provider_google = cls.env.ref("auth_oauth.provider_google")
-        cls.user_model = cls.env["res.users"].with_context(
-            {"tracking_disable": True, "no_reset_password": True}
+@tagged("-at_install", "post_install")
```
Comment (pedrobaeza) [inline auth_oauth_multi_token/tests/test_multi_token.py]: Why you need post-install?

File: auth_oauth_multi_token/tests/test_multi_token.py [hunk 2]
```diff
@@ -4,28 +4,24 @@
...
+class TestMultiToken(TransactionCase):
+    def setUp(self):
```
Comment (pedrobaeza) [inline auth_oauth_multi_token/tests/test_multi_token.py]: And why not setUpClass?

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done!!

File: auth_oauth_multi_token/tests/test_multi_token.py (hunk 1 above)
Comment (pedrobaeza) [reply auth_oauth_multi_token/tests/test_multi_token.py]: What about this? If you need post_install by any reason, please put a comment explaining why for not being removed in later revisions.

**Comment by sergio-teruel** [comment]
Ready!

**Review by pedrobaeza — APPROVED** [review]

**Review by Abranes — APPROVED** [review]
LGTM

**Pull request merged**